src/corefx/System.Globalization.Native/localeStringData.cpp

```diff
@@ -224,6 +229,16 @@ extern "C" int32_t GetLocaleInfoString(const UChar* localeName, LocaleStringData
 		case PositiveInfinitySymbol:
 			status = GetLocaleInfoDecimalFormatSymbol(locale, DecimalFormatSymbols::kInfinitySymbol, value, valueLength);
 			break;
+		case ParentName:
+			if (strlen(locale.getCountry()) > 0)
```

Comment:
No they are not the same. I coded it this way because of unknowns around supported incoming locale strings that specify more than language-region. uloc_getParent parses searching for the _ delimiters so it depends on what we want to support on the incoming locale string and consistency with Windows. ICU and Windows both support language-script-region so I will change the code to use uloc_getParent because that will support 'script' such as in the string "sr-Latn-RS" where script="Latn" (which is a well-known ISO script code). I also need to add tests for this. ICU however also supports locales with a 'variant' such as "en-IE-PREEURO" (variant="PREEURO") or "sr-Latn-RS-REVISED" (variant="REVISED") which are not well-known and Windows doesn't support them but IMO it makes sense to allow these strings to come into ICU and also work with the parent property. FWIW in the next wave of changes I will add validation to the managed side to check for unsupported incoming locale strings such as specifying a calendar, but which use the @ sign, not a _ or - (per the email thread on this topic)